rest/src/main.rs

```diff
@@ -55,14 +55,18 @@ async fn validators() -> Value {
             .await
             .unwrap(); // TODO: cache
 
-        let status = if validator.unbonding {
+        let validator_status = if validator.unbonding {
             "BOND_STATUS_UNBONDING"
         } else if validator.in_active_set {
             "BOND_STATUS_BONDED"
         } else {
             "BOND_STATUS_UNBONDED"
         };
 
+        if !status.is_none() && status != Some(validator_status.to_owned()) {
```

Comment:
this part looks ugly but I am unsure if there's a better way to say "if it's either not set or doesn't equal to whatever status validator has"

Reply:
This isn't a bad way to express that, the only way I can see to simplify would be changing `!status.is_none()` to `status.is_some()`.

Reply:
Just noting that this check is case-sensitive. I'm not sure if the SDK equivalent is or not but if not, it's worth changing to maintain compatibility (can come in a later PR).

Reply:
honestly not sure about that, I only added this as ping.pub expects this endpoint to return only active validators on /validators?status=BOND_STATUS_BONDED
anyways, I guess this one is anyways better than not supporting this param at all right?

Reply:
> Just noting that this check is case-sensitive. I'm not sure if the SDK equivalent is or not

actually somewhat neither lol
```
"rpc error: code = InvalidArgument desc = invalid validator status bond_status_bonded: invalid request"
```

(this is when I query my own node on Cosmos Hub with status as a valid status but in lowercase: https://api.cosmos.quokkastake.io/cosmos/staking/v1beta1/validators?status=bond_status_bonded)